crates/nix_rs/src/copy.rs

```diff
@@ -0,0 +1,19 @@
+use crate::command::{CommandError, NixCmd};
+use std::path::PathBuf;
+
+/// Runs `nix copy` in Rust
```

Comment:
This should be normal function doc without relying on user's knowledge of `nix copy` (which command should be mentioned as sidenote, however).

Reply:
The same applies for other parts of this PR.